src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -94,9 +106,10 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
   submitEnrollData(): void {
     this.isEnrolling = true;
     this.enrollErrorMessage = '';
+    this.allStudentChunks = [];
+
     const newStudentsHOTInstance: Handsontable =
         this.hotRegisterer.getInstance(this.newStudentsHOT);
-
     const hotInstanceColHeaders: string[] = (newStudentsHOTInstance.getColHeader() as string[]);
 
     const studentsEnrollRequest: StudentsEnrollRequest = {
```

Comment:
there is no point nesting here if we are always unpacking - just declare as array